src/pages/api/multipleProductValidity.ts

```diff
@@ -127,7 +127,7 @@ export default async (req: NextApiRequest, res: NextApiResponse): Promise<void>
         };
```

Comment:
Do we not want to add the email and uuid to this decision data (also in periodValidity.ts and productDetails.ts) like we do for the single/return tickets?